bldsys/cmake/sample_helper.cmake

```diff
@@ -135,15 +148,20 @@ function(add_project)
 
     source_group("\\" FILES ${TARGET_FILES})
 
-    # Add shaders to project group
+    # Add GLSL shaders to project group
     if (SHADERS_GLSL)
         source_group("\\Shaders" FILES ${SHADERS_GLSL})
     endif()
 
+    #Add HLSL shaders to project group
+    if (SHADERS_HLSL)
```

Comment:
I see, it works as it is... but shouldn't be `TARGET_SHADERS_HSLS` used here, instead of plain `SHADERS_HSLS`?
The same holds for the SHADERS_GLSL above.

Reply:
Changed to `TARGET_SHADERS_HSLS ` as you suggested.